docs/changelog.rst

```diff
@@ -20,6 +20,7 @@ New features:
 * :doc:`plugins/autobpm`: Add new configuration option ``beat_track_kwargs``
   which enables adjusting keyword arguments supplied to librosa's
   ``beat_track`` function call.
+* New multi-value `genres` tag
```

Comment:
Can you expand a bit here, maybe with an example that shows why it's useful?  The point of such a tag should be made obvious to end users scrolling through.